test/controllers/search_controller_test.rb

```diff
@@ -0,0 +1,43 @@
+require "test_helper"
+
+describe SearchController do
+  it "should get home page" do
+    get root_path
+    must_respond_with :success
+  end
+
+  it "can get recipes index" do
+
+    VCR.use_cassette("recipes") do
+      get recipes_path
+
+      query = "chicken"
+      RecipeApiWrapper.find_recipes(query)
```

Comment:
Why are you making API Wrapper calls directly here?  This should be only making controller calls.